src/main/java/alfio/extension/JSErrorReporter.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * This file is part of alf.io.
+ *
+ * alf.io is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * alf.io is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with alf.io.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package alfio.extension;
+
+import org.mozilla.javascript.ErrorReporter;
+import org.mozilla.javascript.EvaluatorException;
+
+/**
+ *
+ * @author Ram Kulkarni
+ * http://ramkulkarni.com/blog/parsing-javascript-code-using-mozilla-rhino/
+ *
+ * Implements ErrorReporter interface of Rhino and prints syntax errors in the JS script.
+ */
+public class JSErrorReporter implements ErrorReporter {
+
+    @Override
+    public void warning(String message, String sourceName, int line, String lineSource, int lineOffset) {
+        System.out.println("Warning : " + message);
```

Comment:
Use Logger instead of System.out.println

Reply:
done